app/wear/src/main/kotlin/io/github/xxfast/nytimes/wear/screens/story/StoryScreen.kt

```diff
@@ -53,137 +39,145 @@ import io.github.xxfast.nytimes.models.TopStorySection
 import io.github.xxfast.nytimes.screens.story.Loading
 import io.github.xxfast.nytimes.screens.story.StoryState
 import io.github.xxfast.nytimes.screens.story.StoryViewModel
-import kotlinx.coroutines.launch
+import io.github.xxfast.nytimes.wear.screens.navigation.NavigationBox
+import io.github.xxfast.nytimes.wear.theme.NYTimesWearTheme
+import kotlinx.datetime.Instant
 
 @Composable
 fun StoryScreen(
   section: TopStorySection,
   uri: ArticleUri,
   title: String,
-  onBack: () -> Unit,
 ) {
   val viewModel: StoryViewModel = rememberViewModel(StoryViewModel::class) { savedState ->
     StoryViewModel(savedState, section, uri, title)
   }
 
   val state: StoryState by viewModel.states.collectAsState()
 
-  SwipeToDismissBox(onDismissed = onBack) {
+  NavigationBox {
     StoryView(
       state = state,
-      onRefresh = viewModel::onRefresh,
-      onBack = onBack,
-      onSave = viewModel::onSave
+      onSave = viewModel::onSave,
+      columnState = it
     )
   }
 }
 
 @Composable
 fun StoryView(
   state: StoryState,
-  onRefresh: () -> Unit,
+  columnState: ScalingLazyColumnState,
   onSave: () -> Unit,
-  onBack: () -> Unit,
 ) {
```

Comment:
I think its better to keep the `NavigationBox` as an implementation detail within `StoryView` - and avoid passing around this `ScalingLazyColumnState`. Let me know what you think 
```kotlin
@Composable
fun StoryScreen(
  section: TopStorySection,
  uri: ArticleUri,
  title: String,
) {
  val viewModel: StoryViewModel = rememberViewModel(StoryViewModel::class) { savedState ->
    StoryViewModel(savedState, section, uri, title)
  }

  val state: StoryState by viewModel.states.collectAsState()

  StoryView(
    state = state,
    onSave = viewModel::onSave,
  )
}

@Composable
fun StoryView(
  state: StoryState,
  onSave: () -> Unit,
) {
  NavigationBox { columnState ->
    ScalingLazyColumn(
      columnState = columnState,
    )
  }
  ..
}
```